### PR TITLE
ovn-k8s-watcher: Add support for run-time commands.

### PR DESCRIPTION
**Pull request by shettyg**
We can now ask the watcher to exit with the run-time
command:
ovs-appctl -t ovn-k8s-watcher exit

We can list all the vlog modules:
ovs-appctl -t ovn-k8s-watcher vlog/list

We can run-time change the logging level of a particular module:
ovs-appctl -t ovn-k8s-watcher vlog/set overlay:file:DBG

Signed-off-by: Gurucharan Shetty <guru@ovn.org>

### CODE REVIEW

File: bin/ovn-k8s-watcher [hunk 1]
```diff
@@ -51,6 +51,9 @@ def main():
     parser.add_argument('--openstack', action='store_true')
 
     # Handle OVS logging and daemonization arguments.
+    ovs.dirs.RUNDIR = "/var/run/openvswitch"
+    ovs.dirs.LOGDIR = "/var/log/openvswitch"
```
Comment (salv-orlando) [inline bin/ovn-k8s-watcher]: as far as you know is there any distro that uses different paths? I've never seen any, but in case there are these paths should be made configurable.

File: ovn_k8s/watcher/watcher.py [hunk 2]
```diff
@@ -23,6 +27,29 @@
...
+    ovs.unixctl.command_register("exit", "", 0, 0, _unixctl_exit, None)
+    error, unixctl_server = ovs.unixctl.server.UnixctlServer.create(None)
+    if error:
+        ovs.util.ovs_fatal(error, "could not create unixctl server", vlog)
+
+    while True:
+        unixctl_server.run()
+        if exiting:
+            unixctl_server.close()
+            sys.exit()
```
Comment (salv-orlando) [inline ovn_k8s/watcher/watcher.py]: What are the intended semantics of ovs-appctl exit?
This causes an abrupt shutdown, which might kill the processor while in the middle of configuring a logical port.

I see implement a graceful shutdown mechanism as not trivial, so maybe we can leave that for another PR.

**Comment by shettyg** [comment]
Oops. I did make a small update with a --force-push which does handle the case of non package installation. Atleast one of the more common cases.

I do agree that a graceful shutdown is probably a little tricky
